front-end/src/components/player-page/stats/leaderboard-ranking.tsx

```diff
@@ -0,0 +1,30 @@
+import {Tooltip, Typography} from "@mui/material";
+import React from "react";
+import {getNumberWithSuffix} from "../../../common/utils";
+
+interface LeaderboardRankingProps {
+    rank: number
+    leagueAvg: number
+    totalPLayersOnLeaderboard: number
+}
+
+const FIRST = "#FFA500";
+const SECOND = "#85BCBF";
+const THIRD = "#834B24";
+const NTH = "#565554"
+
+const rankToColorMap: Record<number, string> = {
```

Comment:
Tbh just make this a function instead.
```
getColor(rank) {
  switch(rank) {
    case 1:
       return theme.x.first.main;
    .....
  }
```

Reply:
If you want. I don't think its too big of a diff. Might just be easier to look at.

Reply:
Personally i find maps easier to look at so im going to keep it the same